_component_design/time-picker.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Time Picker
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: Enable users to select a specific time or window from a sequential list. The Time Picker is often accompanied by a date picker for the purpose of creating a date range filter.
+preview-image: preview-images/time-picker.svg
+resource: true
+last-modified: 2017-08-17
+---
+
+{% include toc.html %}
+
+## Introduction
```

Comment:
Rather than leave you notes for all instances of H2 or H3 heading level I will just note it here and move on. 

Include either `{: .hxSectionTitle}` or `{: .hxSubSectionTitle}` for each heading to make the class apply the correct spacing.

Reply:
Leaving a note for memory: We're taking these class name calls out per @araiford: https://github.com/rackerlabs/design-system/pull/292#issuecomment-338716258